modules/site-health/admin-site-health.php

```diff
@@ -172,6 +184,43 @@ public function info_options( $debug_info ) {
 		return $debug_info;
 	}
 
+	/**
+	 * Get an array with post_type as key and post IDs as value
+	 *
+	 * @param int $limit  Nb of post max to show per post type.
+	 * @return array
+	 *
+	 * @since   3.0
+	 */
+	public function get_post_ids_without_lang( $limit = 5 ) {
+		$posts     = array();
+		$languages = pll_languages_list();
```

Comment:
Don't use API functions internally (to avoid relying on global in phunit tests). Here you can access to the internal method `$this->model->get_languages_list()`.